### DISCUSSION TITLE
Feature Request: Config to download duckdb extensions throughout https

### DISCUSSION THREAD

**Discussion by mariotaddeucci** [duckdb/duckdb#12137] [Ideas]
Why do you want this feature?
This is a feature to simplify the duckdb adoption in some companies
On my current company by default all http requests are bloqued and only after a firewall rule enabling specific url do be accessed.

Duckdb could have an option to setup to access the endpoint as https, by default keeping in http.
This feature is just to be more compliant with some companies security rules and then becoming easier to run on then.

**Comment by cn00** [comment]
I also meet this issue.

**Comment by carlopi** [comment]
From duckdb v1.5.0, after `LOAD httpfs` all traffic connected to extension install will happen via httpfs (that is https). See https://github.com/duckdb/duckdb/pull/20877

Note that there is a catch: you still need to install (needed once per version) and load httpfs (one per sessions)

**Comment by wernerdaehn** [comment]
Which only has the problem, that you need httpfs to download the httpfs extensions via https.
We are facing the same problem.
